api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -19,8 +20,24 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: QUIC listener config]
 
+// Configuration specific to QUIC crypto stream.
+message QuicCryptoStream {
+  string name = 1 [(validate.rules).string = {min_len: 1}];
+
+  // [#extension-category: envoy.quic.server.crypto_stream]
+  google.protobuf.Any typed_config = 2;
+}
+
+// Configuration specific to QUIC proof source.
+message QuicProofSource {
+  string name = 1 [(validate.rules).string = {min_len: 1}];
+
+  // [#extension-category: envoy.quic.proof_source]
+  google.protobuf.Any typed_config = 2;
```

Comment:
Why doesn't this use `TypedExtensionConfig`? (and above)

Reply:
I didn't know about it. Switched now.